python-checks/src/test/resources/checks/deadStore.py

```diff
@@ -400,3 +400,22 @@ def multiple_issues(a):
     print(b)
 
 
+def nested_packages_import():
```

Comment:
I think it would be clearer to split this into 2 dedicated tests: one for the FP we want to fix, and one for the FN we accept. Here, both are a bit mixed and I think it will be difficult to understand outside the context of this ticket.

I'd suggest having something like:

```
def imports_of_different_submodules_no_fp():
    import sample.first
    import sample.second
    b = sample.first.First()
    c = sample.second.Second()


def nested_packages_imports_fn():
    import sample # FN
    import sample.first # FN
    import sample.first.third

    a = sample.Sample()
    b = sample.first.First()
    d = sample.first.third.Third()
```